unabridged-works-volume-3/original/15-letters.adoc

```diff
@@ -44,13 +45,16 @@ that that may be worn out which hinders the unity, and so,
 ye may feel your King and Saviour exalted upon his throne in your hearts:
 this is your rest, peace, life, kingdom, and crown forever.
 
+[.signed-section-signature]
 I+++.+++ P.
 
-TO +++_______+++ +++_______+++
+[.letter-heading]
+To +++_______+++ +++_______+++
 
-MY DEAR FRIEND,
+[.salutation]
+My Dear Friend,
 
-$$...$$This then is the way of redemption;
++++...+++ This then is the way of redemption;
```

Comment:
When we want ellipses, I try to use this (weird) format instead: `&hellip;`, so can you change this to:

```ASCIIDOC
&hellip;This then is the way of redemption;
```